dm/pkg/checker/privilege.go

```diff
@@ -369,34 +372,36 @@ func VerifyPrivileges(
 						if privs.needGlobal {
 							continue
 						}
-						dbPrivs, ok := privs.dbs[dbName]
-						if !ok || dbPrivs.wholeDB {
-							continue
-						}
-						if _, ok := dbPrivs.tables[tableName]; !ok {
-							continue
+						for dbName, dbPrivs := range privs.dbs {
```

Comment:
> The _ and % wildcards are permitted when specifying database names in [GRANT](https://dev.mysql.com/doc/refman/8.0/en/grant.html) statements that grant privileges at the database level (GRANT ... ON db_name.*).

wildcard cannot be used on table level grant, not need to change here

Reply:
https://github.com/pingcap/tiflow/pull/7739/commits/8dc362cba5951ae397db60748b733c04f042fa5a